backend/tasks/training_packs/task.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import os
+
+from sqlalchemy import desc, func
+
+from backend.database.objects import Playlist, PlayerGame, Game, TrainingPack
+from backend.database.wrapper.query_filter_builder import QueryFilterBuilder
+from backend.tasks.training_packs.training_packs import create_pack_from_replays
+from backend.utils.cloud_handler import upload_training_pack
+
+playlists = [
+    Playlist.UNRANKED_DUELS,
```

Comment:
Maybe put these in a more Central location in case we need this later?
Maybe a file called standard_stadiums?
Or standard_maps
Or standard_modes

Might be useful long term for stats

Might also make sense in carball actually

Reply:
nah, this might change in the future and doesn't need to be used anywhere else for now